### PR TITLE
Update pr_helper criticality score

### PR DESCRIPTION
**Pull request by hogo6002**
Update repo_url as criticality score does not support url ends with '.git'. 
Example: 
~/go/bin$ `./criticality_score --format json -gcp-project-id=clusterfuzz-external https://github.com/kubernetes/kubernetes.git`
2023-06-21 14:27:39.583	INFO	Preparing default scorer
2023-06-21 14:27:41.007	INFO	deps.dev signal source enabled
`2023-06-21 14:27:41.869	WARN	Repo cannot be collected	{"worker": 0, "url": "https://github.com/kubernetes/kubernetes.git", "error": "repo failed: not found: https://github.com/kubernetes/kubernetes.git"}`

### CODE REVIEW

File: infra/pr_helper.py [hunk 1]
```diff
@@ -35,14 +35,21 @@
 
 def get_criticality_score(repo_url):
   """Gets the criticality score of the project."""
+  if repo_url.endswith('.git'):
```
Comment (oliverchang) [inline infra/pr_helper.py:38]: Can you add a comment explaining why this is needed?

File: infra/pr_helper.py [hunk 2]
```diff
@@ -88,7 +95,7 @@ def main():
       repo_url = new_project.get('main_repo')
       if repo_url is None:
         message += (f'{pr_author} is integrating a new project, '
-                    'but the `repo_url` is missing. '
+                    'but the `main_repo` is missing. '
```
Comment (Alan32Liu) [inline infra/pr_helper.py:98]: Will this modification overlap with [this one](https://github.com/google/oss-fuzz/pull/10553/files#diff-8cde736463fcc3019311b0661eed74d119d2103c6f1754dcbeeb50e8f7664733R91)?

File: infra/pr_helper.py [hunk 3]
```diff
@@ -35,14 +35,21 @@
...
       '-gcp-project-id=clusterfuzz-external', '-depsdev-disable', repo_url
   ],
                           capture_output=True,
                           text=True)
 
-  report_dict = json.loads(report.stdout)
+  try:
+    report_dict = json.loads(report.stdout)
+  except:
+    print(f'stdout: {report.stdout}')
```
Comment (oliverchang) [inline infra/pr_helper.py:50]: Can we add "criticality score" to these strings to make it clearer where these stdout/stderr are coming from?

**Review by Alan32Liu — APPROVED** [review]
Thanks!

File: infra/pr_helper.py (hunk 2 above)
Comment (hogo6002) [reply infra/pr_helper.py:98]: I am not sure if we want to make change to the doc file.

**Review by oliverchang — APPROVED** [review]

**Comment by hogo6002** [comment]
Not sure why project tests failed. Re-run doesn't work.

**Comment by Alan32Liu** [comment]
> Not sure why project tests failed. Re-run doesn't work.

I don't think that's related to your PR, please feel free to merge.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**